src/webui/www/private/addtrackers.html

```diff
@@ -1,15 +1,16 @@
 <!DOCTYPE html>
 <html lang="${LANG}">
+
 <head>
     <meta charset="UTF-8" />
     <title>QBT_TR(Trackers addition dialog)QBT_TR[CONTEXT=TrackersAdditionDlg]</title>
     <link rel="stylesheet" href="css/style.css" type="text/css" />
     <script src="scripts/lib/mootools-1.2-core-yc.js"></script>
     <script src="scripts/lib/mootools-1.2-more.js"></script>
     <script>
-        window.addEvent('domready', function(){
+        window.addEvent('domready', function() {
```

Comment:
In c++ we have function start brace in its own line, but since current position is common in JavaScript coding style (AFAIK), I don't object.